src/jukebox/components/battery_monitor/batt_mon_i2c_ina219/__init__.py

```diff
@@ -0,0 +1,94 @@
+# MIT License
```

Comment:
I suggest the whole license block with

```
# RPi-Jukebox-RFID Version 3
# Copyright (c) See file LICENSE in project root folder
```

Reply:
Sorry for being not clear enough.

my suggestion was to replace the **whole** license block with the thin 2-liner. See here for an example https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/src/jukebox/jukebox/multitimer.py

The current text is pretty redundant to the LICENSE file, so we went for a reference only. This has not been changed in all files, though.